src/app/store/transactions/transaction.hooks.ts

```diff
@@ -81,14 +82,14 @@ export function useEstimatedTransactionByteLength() {
 
 export function useSignTransactionSoftwareWallet() {
   const account = useCurrentAccount();
-  if (!account) throw new Error('Cannot sign a transaction without an account');
   return useCallback(
     (tx: StacksTransaction) => {
       const signer = new TransactionSigner(tx);
+      if (!account) return null;
```

Comment:
Could we throw the error here instead? Returning `null` would be quite a silent failure.

Reply:
The typecheck made me add some error logs (and a toast) once the signed tx is returned null. If that isn't good, I can remove those and throw the error here?